Modules/RoleRequestModule.cs

```diff
@@ -15,7 +15,11 @@ namespace DiscordBot.Modules
     public class RoleRequestModule : ModuleBase
     {
         public RoleRequestService RoleRequestService { get; set; }
-        public RoleRequestModule(RoleRequestService roleRequestService) { RoleRequestService = roleRequestService; }
+        public RoleRequestModule(RoleRequestService roleRequestService)
+        {
+            RoleRequestService =
+            roleRequestService;
```

Comment:
Variable assignment on single line
```suggestion
    RoleRequestService = roleRequestService
```